app/controllers/structures_controller.rb

```diff
@@ -51,4 +59,10 @@ def display_error(messages)
   def decrypted_token
     Argo.verifier.verified(params[:item_id], purpose: :view_token)
   end
+
+  def fetch_cocina(item_id:, user_version_id: nil)
```

Comment:
I wonder if we need these params passed from above, given `#item_id` and `#user_version_id` can be called from `#fetch_cocina` without being passed in.

Reply:
Good catch, I think I had this in place before cleaning up the methods above.